datumaro/cli/commands/download.py

```diff
@@ -0,0 +1,101 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import argparse
+import logging as log
+import os
+import os.path as osp
+
+from datumaro.components.extractor_tfds import (
+    AVAILABLE_TFDS_DATASETS, TFDS_EXTRACTOR_AVAILABLE, make_tfds_extractor,
+)
+from datumaro.components.project import Environment
+from datumaro.util.os_util import make_file_name
+
+from ..util import MultilineFormatter
+from ..util.errors import CliException
+from ..util.project import generate_next_file_name
+
+
+def build_parser(parser_ctor=argparse.ArgumentParser):
+    builtin_writers = sorted(Environment().converters)
+    if TFDS_EXTRACTOR_AVAILABLE:
+        available_datasets = ", ".join(
+            f'tfds:{name}' for name in AVAILABLE_TFDS_DATASETS)
+    else:
+        available_datasets = "N/A (install TensorFlow Datasets " \
+            "to download datasets)"
+
+    parser = parser_ctor(
+        help="Download a publicly available dataset and save it "
+            "in a given format",
+        description="""
+        TBD.|n
+        |n
+        Supported datasets: {}|n
+        |n
+        Supported output formats: {}|n
+        |n
+        Examples: TBD
+        """.format(available_datasets, ', '.join(builtin_writers)),
+        formatter_class=MultilineFormatter)
+
+    parser.add_argument('-i', '--dataset-id', required=True,
```

Comment:
What do you think about an option to display dataset info by dataset id?

Reply:
What kind of info do you have in mind?

Either way, I think that as long as we only get datasets from TFDS, it would be more practical to link to the [TFDS catalog](https://www.tensorflow.org/datasets/catalog/overview) from the help text, which contains all the info one could want.

Reply:
Yeah, I think, it mostly covers what I was thinking about. Info like dataset domain, annotation types, subsets, classes, size etc.

Reply:
I linked it in the help message.